src/components/elements/accordion.tsx

```diff
@@ -109,6 +109,7 @@ type ChildType = {
         plus?: boolean
         arrow_thin?: boolean
         is_showed?: boolean
+        classname?: string
```

Comment:
It's fine to use `className` as `props` and we don't need to rename it to `classname`.
But if we are already using `className` for another purpose then we can use other names such as `styleName` or `childStyle`. @prince-deriv can suggest the best approach in here.

Reply:
`className` as props should work specially if it's our custom components.